internal/commands/untrust_builder.go

```diff
@@ -17,6 +17,7 @@ func UntrustBuilder(logger logging.Logger, cfg config.Config) *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
 			builderName := args[0]
+
 			existingBuilders := cfg.TrustedBuilders
```

Comment:
It doesn't relate to your PR but maybe it's better to change the variable name from `existingBuilders` to `existingTrustedBuilders`.